src/routes/terms.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+const fs = require('fs-extra');
+const { Router } = require('express');
+const { marked } = require('marked');
+const matter = require('gray-matter');
+const logger = require('../logger');
+
+const router = new Router();
+
+router.get('/:slug', async (request, response, next) => {
+	const renderData = {
+		layout: 'blog-opengraph'
+	};
+
+	const termName = request.params.slug;
+
+	let rawTerm;
+	try {
+		rawTerm = await fs.readFile(path.join('terms', `${termName}.md`), 'utf-8');
```

Comment:
path traversal?

Reply:
Good catch, this is actually in the blog posts too